src/tooltip/tooltip.ts

```diff
@@ -107,6 +107,14 @@ export class NgbTooltip implements OnInit, OnDestroy {
    * @since 1.1.0
    */
   @Input() disableTooltip: boolean;
+  /**
+   * Delay (in ms) before opening the tooltip after a non-manual opening trigger.
+   */
+  @Input() triggerOpenDelay = 0;
```

Comment:
- these inputs should come from `NgbTooltipConfig`
- related tests are in `ngb-tooltip-config.ts`